src/main/java/org/gridsuite/modification/server/entities/equipment/creation/SubstationCreationEntity.java

```diff
@@ -63,5 +70,6 @@ public SubstationCreationInfos toSubstationCreationInfos() {
                 .equipmentName(getEquipmentName())
                 .substationCountry(getCountry()).properties(getProperties());
     }
+
```

Comment:
It lacks a
```
@Override
public void cloneWithIdsToNull() {
    super.cloneWithIdsToNull();
    if (this.properties != null) {
        this.properties = new HashMap<>(properties);
    }
}
```
for (forgotten) test for properties in matcher to succeed

Reply:
An another mean, as it seems cloneWithIdsToNull is part of the legacy interface we do not want to use anymore.

Reply:
I added it myself. The trick around JPA, with this cloneWithIdsToNull will have to be re-evaluated.